config/kuadrant/kustomization.yaml

```diff
@@ -1,7 +1,7 @@
 resources:
-- github.com/Kuadrant/kuadrant-operator/config/deploy?ref=gateway-api-v0.6.2
+- github.com/Kuadrant/kuadrant-operator/config/deploy?ref=v0.3.1
```

Comment:
Kuadrant is now being handled by ocm via addons so its not needed anymore

Reply:
yeah, when your PR lands I'll rebase and remove this. Still want the module update.